libs/core/cornflow_core/exceptions/exceptions.py

```diff
@@ -4,7 +4,9 @@
 """
 from flask import jsonify
 from webargs.flaskparser import parser
+from werkzeug.exceptions import InternalServerError
```

Comment:
`InternalServerError` is not used anywhere?

Reply:
You're right